### ISSUE TITLE
WSL network does not work

### ISSUE THREAD

**Issue by fernandodandrea** [microsoft/WSL#8601]
### Version

Microsoft Windows [Version 10.0.19043.1826]

### WSL Version

- [X] WSL 2
- [ ] WSL 1

### Kernel Version

5.10.102.1

### Distro Version

Ubuntu 20.04 and Manjaro 

### Repro Steps

Run `WSL`
Try to access any command that requires network: `ping`, `code /etc/resolv.conf`, or `sudo pacman -Syyu`.

### Expected Behavior

Network should be accessible.

### Actual Behavior

`ping 8.8.8.8` returns `ping: connect: Network is unreachable`.

### Diagnostic Logs

Output of `ip addr`:
```
1: lo: <LOOPBACK> mtu 65536 qdisc noop state DOWN group default qlen 1000
    link/loopback 00:00:00:00:00:00 brd 00:00:00:00:00:00
2: bond0: <BROADCAST,MULTICAST,MASTER> mtu 1500 qdisc noop state DOWN group default qlen 1000
    link/ether 32:fd:e4:90:75:42 brd ff:ff:ff:ff:ff:ff
3: dummy0: <BROADCAST,NOARP> mtu 1500 qdisc noop state DOWN group default qlen 1000
    link/ether ea:10:ed:95:34:09 brd ff:ff:ff:ff:ff:ff
4: eth0: <BROADCAST,MULTICAST> mtu 1500 qdisc noop state DOWN group default qlen 1000
    link/ether 00:15:5d:9f:29:b8 brd ff:ff:ff:ff:ff:ff
5: tunl0@NONE: <NOARP> mtu 1480 qdisc noop state DOWN group default qlen 1000
    link/ipip 0.0.0.0 brd 0.0.0.0
6: sit0@NONE: <NOARP> mtu 1480 qdisc noop state DOWN group default qlen 1000
    link/sit 0.0.0.0 brd 0.0.0.0

```
Output of `ip route`:
```
Error: ipv4: FIB table does not exist.
Dump terminated
```

Output of `cat /etc/resolv.conf`:
```
# This file was automatically generated by WSL. To stop automatic generation of this file, add the following entry to /etc/wsl.conf:
# [network]
# generateResolvConf = false
nameserver 172.24.48.1
```

Output of `ping 172.24.48.1`:
`ping: connect: Network is unreachable`

I also tried to entirely remove hyper-v and WSL and then reinstall. I also tried to set WSL2 vEthernet as bridge mode. Output of `type %userprofile%\.wslconfig`:
```
[wsl2]
networkingMode=bridged
vmSwitch=wsl-switch
```

**Comment by kse-systematic** [comment]
Same issue here

**Comment by momvart** [comment]
Same issue here with the same output as the OP has included. Is this a problem with the new version of Ubuntu?

**Comment by fernandodandrea** [comment]
> Same issue here with the same output as the OP has included. Is this a problem with the new version of Ubuntu?

No, it is not, for I'm also having it on Manjaro.

**Comment by momvart** [comment]
I'm not sure if this is related or just has happened accidentally, but today when I started bash and executed `sudo apt update` it could check for updates, and now its network is set up again (has obtained IP, successfully pings, etc.).

**Comment by fernandodandrea** [comment]
> I'm not sure if this is related or just has happened accidentally, but today when I started bash and executed `sudo apt update` it could check for updates, and now its network is set up again (has obtained IP, successfully pings, etc.).

Nothing new here.

**Comment by fernandodandrea** [comment]
I got some new information. I tried to uninstall and reinstall all the VM platform and WSL after removing OpenVPN, Virtual Box, etc, everything that messed up with network adaptors, because whenever I try to diagnose the vEthernet adaptor, the troubleshooter gives me an **error code 0x80004005** before it can actually run. This persists even after the said full reinstall, and have a hunch the problem is related to this.
